WordPress/Classes/ViewRelated/SuggestionsListViewModel.swift

```diff
@@ -123,15 +123,42 @@ import CoreData
         }
         switch suggestions {
         case .users(let userSuggestions):
+            let sortedUserSuggestions = Self.sort(
+                userSuggestions: userSuggestions, by: searchQuery
+            )
             return Self.searchResultByMovingProminentSuggestionsToTop(
-                userSuggestions: userSuggestions,
+                userSuggestions: sortedUserSuggestions,
                 prominentSuggestionsIds: prominentSuggestionsIds
             )
         case .sites(let siteSuggestions):
             return .sites(siteSuggestions)
         }
     }
 
+    /// Sort user suggestions by prefix first, then alphabetically. The collection is sorted first by checking if displayName begins with the provided prefix. The remaining items are sorted alphabetically by displayName. The prefix comparison is both case-insensitive and diacritic-insensitive.
```

Comment:
```suggestion
    /// Sort user suggestions by prefix first, then alphabetically. The collection is sorted first by checking if username or displayName begins with the provided prefix. The remaining items are sorted alphabetically by displayName. The prefix comparison is both case-insensitive and diacritic-insensitive.
```

Reply:
Yes, I forgot to update the docs. Thanks for noticing it.